ngui/ui/src/components/forms/ConnectCloudAccountForm/FormElements/AwsConnectionForm/AwsConnectionFormElements.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { ReactNode, useState } from "react";
+import Alert from "@mui/material/Alert";
+import Box from "@mui/material/Box";
+import Link from "@mui/material/Link";
+import Typography from "@mui/material/Typography";
+import { FormattedMessage } from "react-intl";
+import ButtonGroup from "../../../../ButtonGroup";
```

Comment:
We usually use an absolute path when importing a "common" component

```suggestion
import ButtonGroup from "components/ButtonGroup";
```

Reply:
done